tests/core/test_looper.py

```diff
@@ -104,3 +104,51 @@ def test_looper_refresh_iterator(enaml_qtbot, enaml_sleep):
     expected = ['{} {}'.format(*it) for it in enumerate(reversed(data))]
     assert [c.text for c in tester.container.children
             if isinstance(c, Label)] == expected
+
+
+@pytest.mark.skipif(not is_qt_available(), reason='Requires a Qt binding')
```

Comment:
I would like to see this test also test the case of passing an iterator since this is the case we are addressing. Also Do we really need Qt for testing this ? I feel like we should be able to validate the logic without a backend (but I may be missing something).